java/rocksjni/write_batch_with_index.cc

```diff
@@ -539,6 +539,23 @@ void Java_org_rocksdb_WBWIRocksIterator_status0(
   rocksdb::RocksDBExceptionJni::ThrowNew(env, s);
 }
 
+/*
+ * Class:     org_rocksdb_WBWIRocksIterator
+ * Method:    refresh0
+ * Signature: (J)V
+ */
+void Java_org_rocksdb_WBWIRocksIterator_refresh0(
+    JNIEnv* env, jobject jobj, jlong handle) {
+  auto* it = reinterpret_cast<rocksdb::Iterator*>(handle);
+  rocksdb::Status s = it->Refresh();
+
+  if (s.ok()) {
+    return;
+  }
+
+  rocksdb::RocksDBExceptionJni::ThrowNew(env, s);
```

Comment:
@sagar0 if you delete line 549 to line 554, and only leave 556 with a NotSupported message, I can merge it.
We need you advice on how we should do with it.